src/Exception/HydratingException.php

```diff
@@ -24,7 +24,24 @@ public function getErrorsMap() { return $this->errorsMap; }
      */
     public function __construct($errorsMap, $message = "", $code = 412)
     {
+        $errorMessages = $this->getErrorMessage($errorsMap);
+        $message .= "Detailed errors : \n " . implode(", ", $errorMessages);
```

Comment:
```suggestion
        $message .= " - Detailed errors : \n " . implode(", ", $errorMessages);
```

Maybe we could add a "dash" before the "Detailed errors" message.